### PR TITLE
[CI]Move CPU inductor test runners and cases to save cost

### PR DESCRIPTION
**Pull request by zxd1997066**
For CPU, only SPR has native support for AMP BF16.

Ref: pytorch/pytorch#138476

### CODE REVIEW

**Comment by huydhn** [comment]
@zxd1997066 Could you help provide some of the context of why this is needed? A CI PR without any description is hard to review

**Comment by zxd1997066** [comment]
> @zxd1997066 Could you help provide some of the context of why this is needed? A CI PR without any description is hard to review

Thanks for the comment, added the description.

**Comment by chuanqi129** [comment]
@zxd1997066 please resolve the code conflict firstly

**Comment by desertfire** [comment]
@huydhn or @ZainRizvi probably have already talked to you guys, but the context is we are trying to cut the Inductor CI expense. This PR will further increase the cost. Fine if this is short-term and you will eventually move them to a different CI tier or CI infra. Just want to bring that for awareness.

**Review by jgong5 — CHANGES_REQUESTED** [review]
@zxd1997066 Can you share the motivation of this change?

**Comment by zxd1997066** [comment]
> @zxd1997066 Can you share the motivation of this change?

yes, for CPU, only SPR(linux.16xlarge.spr) has native support for AMP BF16, cascade Lake (linux.12xlarge) does not have.

**Review by jgong5 — APPROVED** [review]

File: .github/workflows/inductor-periodic.yml
Comment (chuanqi129) [inline .github/workflows/inductor-periodic.yml:130]: This is build job, we need add test job in here also

File: .github/workflows/inductor.yml
Comment (chuanqi129) [inline .github/workflows/inductor.yml:122]: Let's check whether we can move some fp32 test into inductor-periodic also

**Comment by huydhn** [comment]
Looking at the change right now, `linux.8xlarge.amx` is cheaper than both `linux.12xlarge` and `linux.16xlarge.spr`, so cost-wise, this is better I think

**Review by zxiiro — APPROVED** [review]
+1 to moving to cheaper systems!

File: .github/workflows/inductor-periodic.yml
Comment (zxiiro) [inline .github/workflows/inductor-periodic.yml:129]: Fix the linter issue.

```suggestion
    secrets:
```

**Comment by chuanqi129** [comment]
> Looking at the change right now, `linux.8xlarge.amx` is cheaper than both `linux.12xlarge` and `linux.16xlarge.spr`, so cost-wise, this is better I think

Hi @huydhn , can we create a new label `ciflow/inductor-periodic`? So that we can test the workflow change in this PR in advance before the landing

**Comment by zxiiro** [comment]
> > Looking at the change right now, `linux.8xlarge.amx` is cheaper than both `linux.12xlarge` and `linux.16xlarge.spr`, so cost-wise, this is better I think
> 
> Hi @huydhn , can we create a new label `ciflow/inductor-periodic`? So that we can test the workflow change in this PR in advance before the landing

Looks like I was able to add the label. I see some jobs starting now so I hope it works as expected.

**Comment by huydhn** [comment]
Anyone with write access to the repo can able to create the label.  Thank @zxiiro for picking it up!

**Review by chuanqi129 — APPROVED** [review]

**Comment by chuanqi129** [comment]
Thanks @huydhn and @zxiiro. The PR and test results LGTM. But there is an unrelated failure. @zxiiro please help to merge it if you don't have further comments

**Comment by zxiiro** [comment]
Looks like we need some additional approvals first.

**Review by desertfire — APPROVED** [review]
Stamping. I think .github/workflows/inductor.yml can be further trimmed.

**Pull request closed without merging**
